hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneDirectory.java

```diff
@@ -1,183 +0,0 @@
-
-    // create sub-dirs under same parent
-    Path subDir5 = new Path("/d1/d2/d3/d4/d5");
-    fs.mkdirs(subDir5);
-    Path subDir6 = new Path("/d1/d2/d3/d4/d6");
-    fs.mkdirs(subDir6);
-    long d5ObjectID = verifyDirKey(d4ObjectID, "d5",
-            "/d1/d2/d3/d4/d5", dirKeys, omMgr);
-    long d6ObjectID = verifyDirKey(d4ObjectID, "d6",
-            "/d1/d2/d3/d4/d6", dirKeys, omMgr);
-    Assert.assertTrue("Wrong objectIds for sub-dirs[" + d5ObjectID +
-                    "/d5, " + d6ObjectID + "/d6] of same parent!",
-            d5ObjectID != d6ObjectID);
-
-    Assert.assertEquals("Wrong OM numKeys metrics",
-            6, cluster.getOzoneManager().getMetrics().getNumKeys());
-  }
-
-  /**
-   * Verify key name format and the DB key existence in the expected dirKeys
-   * list.
-   *
-   * @param key     table keyName
-   * @param dirKeys expected keyName
-   */
-  private void verifyKeyFormat(String key, ArrayList<String> dirKeys) {
-    String[] keyParts = StringUtils.split(key,
-            OzoneConsts.OM_KEY_PREFIX.charAt(0));
-    Assert.assertEquals("Invalid KeyName", 2, keyParts.length);
-    boolean removed = dirKeys.remove(key);
-    Assert.assertTrue("Key:" + key + " doesn't exists in directory table!",
-            removed);
-  }
-
-  long verifyDirKey(long parentId, String dirKey, String absolutePath,
-                    ArrayList<String> dirKeys, OMMetadataManager omMgr)
-          throws Exception {
-    String dbKey = parentId + "/" + dirKey;
-    dirKeys.add(dbKey);
-    OmDirectoryInfo dirInfo = omMgr.getDirectoryTable().get(dbKey);
-    Assert.assertNotNull("Failed to find " + absolutePath +
-            " using dbKey: " + dbKey, dirInfo);
-    Assert.assertEquals("Parent Id mismatches", parentId,
-            dirInfo.getParentObjectID());
-    Assert.assertEquals("Mismatches directory name", dirKey,
-            dirInfo.getName());
-    Assert.assertTrue("Mismatches directory creation time param",
-            dirInfo.getCreationTime() > 0);
-    Assert.assertEquals("Mismatches directory modification time param",
-            dirInfo.getCreationTime(), dirInfo.getModificationTime());
-    Assert.assertEquals("Wrong representation!",
-            dbKey + ":" + dirInfo.getObjectID(), dirInfo.toString());
-    return dirInfo.getObjectID();
-  }
-
-  private void setupOzoneFileSystem()
-          throws IOException, TimeoutException, InterruptedException {
-    OzoneConfiguration conf = new OzoneConfiguration();
-    conf.setInt(FS_TRASH_INTERVAL_KEY, 1);
-    TestOMRequestUtils.configureFSOptimizedPaths(conf,
-            true, OMConfigKeys.OZONE_OM_METADATA_LAYOUT_PREFIX);
-    cluster = MiniOzoneCluster.newBuilder(conf)
-            .setNumDatanodes(3)
-            .build();
-    cluster.waitForClusterToBeReady();
-    // create a volume and a bucket to be used by OzoneFileSystem
-    OzoneBucket bucket = TestDataUtil.createVolumeAndBucket(cluster);
-    volumeName = bucket.getVolumeName();
```

Comment:
How is this tested now ? with the test removed ?

Reply:
Thanks @mukul1987 for the reviews. 

I've moved `TestOzoneDirectory#testMultiLevelDirs` and `TestOzoneFileOps#testCreateFile` to `TestOzoneFileSystemWithFSO.java` test class. Both were starting/stopping cluster and taking nearly ~25secs each, total ~50secs. IMHO, these two are running on `MiniOzoneCluster` and can be tested using TestOzoneFileSystemWithFSO or any other class which starts `MiniOzoneCluster`, where its finishing the each run in milliseconds (<100 milliseconds in my local env).

Reply:
ok sure, LGTM